faker/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+mock==1.0.1
```

Comment:
I think `tests/requirements.txt` is the way to go.

The various `travis.yml` and co. will need to be updated to run `pip install -r tests/requirements.txt`